### ISSUE TITLE
Switch step 4 and step 5?

### ISSUE THREAD

**Issue by SimonLab** [dwyl/phoenix-liveview-counter-tutorial#7]
If we update the `endpoint.ex` file before the `router.ex` file we won't need to add
```ex
import Phoenix.LiveView.Router
```
in the router file as it will already be imported automatically with the new change from the endoint file:
 
```ex
def router do
  quote do
    ...
  import Phoenix.LiveView.Router
  end
end
```

**Comment by nelsonic** [comment]
@SimonLab good observation. 👍 
But then perhaps we can simply _remove_ the first two lines of Step 4 
given that the `Phoenix.LiveView.Router` is already imported in `endpoint.ex`.

**Comment by nelsonic** [comment]
Included in #6 :shipit:

**Comment by SimonLab** [comment]
> But then perhaps we can simply remove the first two lines of Step 4

Some editors (in my case VScode) display an error if we don't import Phoenix.LiveView.Router on step 4. The error will be gone as soon as step 5 is finished but I'm wondering if it will confuse people.
we could add a warning on step 4 or we could merge step 4 and 5 together?

**Comment by nelsonic** [comment]
Ah .... ok. 
If your editor is complaining, which is impressive that it _understands_ the lack of the import, 
then let's make the switch!

**Comment by nelsonic** [comment]
Change included in PR #6 👍

**Comment by SimonLab** [comment]
resolved with PR #6
